lib/matplotlib/testing/jpl_units/UnitDbl.py

```diff
@@ -116,7 +116,7 @@ def __nonzero__( self ):
       = RETURN VALUE
       - Returns true if the value is non-zero.
       """
-      return self._value.__nonzero__()
+      return self._value.__bool__()
```

Comment:
This used to fail in python 3 becasue `__nonzero__` isn't defined in python 3; `__bool__` works for both 2 and 3.